src/role/mod.rs

```diff
@@ -0,0 +1,96 @@
+mod aria_abstract_roles;
```

Comment:
https://github.com/A11yance/aria-query/blob/b0bcd43afecd266d73dd710fbf744836c55a4c0a/src/rolesMap.js#L18-L44

I implement the converter logic originated from the original aria-query later